vault.md

```diff
@@ -155,7 +155,7 @@ Then tape `c` in order to show the Dev Services containers running. You should s
 
 ```
 
-If you want to use this Vault in command line for creating or reading secrets, you can configure the address and token as follows:
+If you want to it using the vault client within a terminal, export the following ENV variables:
```

Comment:
That should be `If you want to use the vault client ...` as the text ``If you want to it using the vault client ...` is not correct